doc/1/getting-started/java/index.md

```diff
@@ -0,0 +1,149 @@
+
+### Ivy:
+
+```html
+<dependency org='io.kuzzle' name='sdk-jvm' rev='1.0.0'>
+  <artifact name='sdk-jvm' ext='pom' ></artifact>
+</dependency>
+```
+
+## First connection
+
+Initialize a new Java project, create a `GettingStartedFirstConnection.java` file and start by adding the code below:
+
+<<< ./snippets/firstconnection.java
+
+This program initializes the Kuzzle Server storage by creating a index, and a collection inside it
+Run the program with the following command:
+
+```bash
+$ javac -classpath ./path/to/the/sdk.jar GettingStartedFirstConnection.java
+$ java -classpath .:./path/to/the/sdk.jar GettingStartedFirstConnection
+Connected!
+Index nyc-open-data created!
+Collection yellow-taxi created!
+```
+
+Congratulations, you performed your first connection to Kuzzle Server via a Java program.
+You now know how to:
+
+- Instantiate Kuzzle SDK and connect to Kuzzle Server using a specific protocol (here `websocket`)
+- Create a index
+- Create a collection within an existing index
+
+## Create your first document
+
+Now that you successfully connected to your Kuzzle Server instance, and created an index and a collection, it's time to manipulate some data.
+
+Here is how Kuzzle structures its storage space:
+
+- indexes contain collections
+- collections contain documents
+  Create a `GettingStartedStorage.java` file in the playground and add this code:
+
+<<< ./snippets/document.java
+
+As you did before, build and run your program:
+
+```bash
+$ javac -classpath ./path/to/the/sdk.jar  GettingStartedStorage.java
+$ java -classpath .:./path/to/the/sdk.jar GettingStartedStorage
+Connected!
+New document added to yellow-taxi collection!
+```
+
+You can perform other actions such as [delete](/sdk/jvm/1/controllers/document/delete), [replace](/sdk/jvm/1/controllers/document/replace) or [search](/sdk/jvm/1/controllers/document/search) documents. There are also other ways to interact with Kuzzle like our [Admin Console](/core/2/guides/essentials/admin-console), the [Kuzzle HTTP API](/core/2/api/essentials/connecting-to-kuzzle) or by using your [own protocol](/core/2/protocols/essentials/getting-started).
+
+Now you know how to:
+
+- Store documents in a Kuzzle Server, and access those
+
+## Subscribe to realtime document notifications (pub/sub)
+
+Time to use realtime with Kuzzle. Create a new file `GettingStartedRealtime.java` with the following code:
+
+<<< ./snippets/realtime.java
+
+This program subscribes to changes made to documents with a `license` field set to `B`, within the `yellow-taxi` collection. Whenever a document matching the provided filters changes, a new notification is received from Kuzzle.
+
+Build and run your program:
+
+```bash
+$ javac -classpath ./path/to/the/sdk.jar GettingStartedRealtime.java
+$ java -classpath .:./path/to/the/sdk.jar GettingStartedRealtime
+Connected!
+Successfully subscribing!
+New document added to yellow-taxi collection!
+New created document notification: [Document content as ConcurrentHashMap]
```

Comment:
hm... can the snippet serialize the document content and print it so that example gives a view of the actual notification content?